server/backend/mlx_runner.py

```
@@ -0,0 +1,1128 @@
+        self._reasoning_end = None
+        self._final_start = None
+        self._model_loaded = False
+
+        # Force garbage collection and clear MLX cache
+        import gc
+
+        gc.collect()
+        try:
+            mx.clear_cache()
+        except Exception:
+            pass  # Continue cleanup even if cache clear fails
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)